### ISSUE TITLE
Segmentation fault with 2.34.0

### ISSUE THREAD

**Issue by tanguydatacamp** [phrase/phrase-cli#157]
Hello 👋 

I have a fresh installation of Phrase-cli (`brew install phrase-cli`) and cannot make it work.

My colleagues, using the same token as me but with an older version of Phrase are able to push/pull.

Here is the issue I encounter:

```bash
$ phrase pull     
            
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x50 pc=0x1050bf8d8]

goroutine 1 [running]:
github.com/phrase/phrase-cli/cmd/internal.Targets.GetAllLocalesCacheKeys(...)
	github.com/phrase/phrase-cli/cmd/internal/pull_target.go:21
github.com/phrase/phrase-cli/cmd/internal.GetLocalesCache(0x14000326000, {0x1053864c0, 0x140002d8858}, {0x0, 0x0})
	github.com/phrase/phrase-cli/cmd/internal/shared.go:39 +0x48
github.com/phrase/phrase-cli/cmd/internal.(*PullCommand).Run(0x14000035c70, 0x10516db74?)
	github.com/phrase/phrase-cli/cmd/internal/pull.go:61 +0x150
github.com/phrase/phrase-cli/cmd.initPull.func1(0x14000300600?, {0x10516d7ad?, 0x4?, 0x10516d741?})
	github.com/phrase/phrase-cli/cmd/pull.go:25 +0xb0
github.com/spf13/cobra.(*Command).execute(0x14000308008, {0x1056f0760, 0x0, 0x0})
	github.com/spf13/cobra@v1.0.0/command.go:846 +0x548
github.com/spf13/cobra.(*Command).ExecuteC(0x1056c2a80)
	github.com/spf13/cobra@v1.0.0/command.go:950 +0x310
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.0.0/command.go:887
github.com/phrase/phrase-cli/cmd.Execute()
	github.com/phrase/phrase-cli/cmd/root.go:70 +0x24
main.main()
	github.com/phrase/phrase-cli/main.go:8 +0x1c
```

Here is phrase info
```
Phrase Strings client version:            2.34.0
Phrase Strings client revision:           DEV
Phrase Strings library revision:          DEV
Last change at:                   LIVE
Go version:                       go1.23.3
```

System info: MacOS Sonoma 14.7.1

**Comment by jablan** [comment]
@tanguydatacamp thanks for reporting. We will investigate, in the meantime, please use the previous version.

Could you please post a minimal configuration file resulting with the error (feel free to anonymize concrete data in it beforehand)?

**Comment by tanguydatacamp** [comment]
> @tanguydatacamp thanks for reporting. We will investigate, in the meantime, please use the previous version.
> 

I can confirm that it works with v2.33.1 :-) 

> Could you please post a minimal configuration file resulting with the error (feel free to anonymize concrete data in it beforehand)?

I honestly have no idea how to do that 🙈  I am using this for the first time. Here is, however the `.phrase.yml` we use

```
phrase:
  host: https://api.us.app.phrase.com/v2
  file_format: nested_json

  push:
    sources:
      - file: ./src/languages/<locale_code>/translation.json
        project_id: 'redacted'
        params:
          autotranslate: true
          update_translations: true
  pull:
    targets:
      - file: ./src/languages/<locale_code>/translation.json
        project_id: 'redacted'
      # common project
      - file: ./src/languages/<locale_code>/common.json
        project_id: 'redacted'
      # glossary project
      - file: ./src/languages/<locale_code>/glossary.json
        project_id: 'redacted'
```

**Comment by jablan** [comment]
hey @tanguydatacamp would you mind trying out newly released v2.34.1?

**Comment by tanguydatacamp** [comment]
That fixes this issue! 🎉 
Many thanks!
